src/controller/model/device.ts

```diff
@@ -515,11 +509,6 @@ class Device extends Entity {
     }
 
     private async interviewInternal(): Promise<void> {
-        // Clear defaultSendWhenActive mode when running interview
-        for (const endpoint of this._endpoints) {
-            endpoint.defaultSendWhenActive = false;
-        }
```

Comment:
It's nice to get rid of this, which was an ugly workaround for interview unittests failing under sendWhenActive.